### PR TITLE
[15.0][MIG] analytic_schedule

### PR DESCRIPTION
**Pull request by DhavalTforgeFlow**

### CODE REVIEW

File: analytic_schedule/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
```
Comment (AaronHForgeFlow) [inline analytic_schedule/__init__.py]: Remove this

File: analytic_schedule/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
```
Comment (AaronHForgeFlow) [inline analytic_schedule/__manifest__.py]:
```suggestion
# Copyright 2017-23 ForgeFlow S.L.
```

File: analytic_schedule/model/__init__.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
```
Comment (AaronHForgeFlow) [inline analytic_schedule/model/__init__.py]: remove this also

File: analytic_schedule/model/project.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
```
Comment (AaronHForgeFlow) [inline analytic_schedule/model/project.py]:
```suggestion
# Copyright 2017-23 ForgeFlow S.L.
```

File: analytic_schedule/views/analyltic_account_view.xml [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 Eficent Business and IT Consulting Services S.L.
```
Comment (AaronHForgeFlow) [inline analytic_schedule/views/analyltic_account_view.xml]: ForgeFlow

**Review by AaronHForgeFlow — COMMENTED** [review]
LG, minor comments

**Review by JuanyDForgeflow — APPROVED** [review]

**Review by AaronHForgeFlow — COMMENTED** [review]
same problem as here: https://github.com/ForgeFlow/eficent-odoo-addons/pull/276#discussion_r1147342479

**Review by AaronHForgeFlow — APPROVED** [review]
Just fixed date propagation and test added

**Pull request merged**